### PR TITLE
Server spoof enhancements

### PR DESCRIPTION
**Pull request by ThebestkillerTBK**
#2604 - Spoof resource pack without brand
#2404 - Channel spoof

### CODE REVIEW

**Comment by GreemDev** [comment]
CI failed: 
``` 
* What went wrong:
A problem occurred configuring root project 'meteor-client'.
> Could not resolve all dependencies for configuration ':modCompileOnly'.
   > Could not resolve io.vram:canvas-fabric-mc119:1.0.+.
     Required by:
         project :
      > Failed to list versions for io.vram:canvas-fabric-mc119.
         > Unable to load Maven meta-data from https://maven.vram.io/io/vram/canvas-fabric-mc119/maven-metadata.xml.
            > Could not GET 'https://maven.vram.io/io/vram/canvas-fabric-mc119/maven-metadata.xml'.
               > maven.vram.io: No address associated with hostname
```

Seems like maybe vram.io changed domains? or they forgot to renew the domain?

**Comment by StormyBytes** [comment]
> ~Seems like maybe vram.io changed domains? or they forgot to renew the domain?~ just checked, it's loading fine, so maybe some weird DNS issue?

CI seems fine now.

File: src/main/java/meteordevelopment/meteorclient/systems/modules/misc/ServerSpoof.java [hunk 1]
```diff
@@ -60,31 +89,46 @@ private void onPacketSend(PacketEvent.Send event) {
...
             Identifier id = packet.getChannel();
 
-            if (id.equals(CustomPayloadC2SPacket.BRAND)) {
+            if (spoofBrand.get() && id.equals(CustomPayloadC2SPacket.BRAND))
                 packet.setData(new PacketByteBuf(Unpooled.buffer()).writeString(brand.get()));
-            }
-            else if (StringUtils.containsIgnoreCase(packet.getData().toString(StandardCharsets.UTF_8), "fabric") && brand.get().equalsIgnoreCase("fabric")) {
-                event.cancel();
+
+            if (channels.get()) for (String channel : channelsOut.get()) {
```
Comment (MineGame159) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/misc/ServerSpoof.java:95]: Please don't use this scuffed formatting, put the for loop inside {} on a new line.

**Comment by MineGame159** [comment]
Why even attempt to block incomming channels? You are just blocking potential features that way.

**Comment by seasnail8169** [comment]
yeah this pr is kinda retarded

**Pull request merged**